packages/twenty-server/src/engine/twenty-orm/pg-shared-pool/__tests__/pg-shared-pool.service.spec.ts

```diff
@@ -0,0 +1,351 @@
+import { Logger, LogLevel } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+
+import { Pool } from 'pg';
+
+import { TwentyConfigService } from 'src/engine/core-modules/twenty-config/twenty-config.service';
+import { PgPoolSharedService } from 'src/engine/twenty-orm/pg-shared-pool/pg-shared-pool.service';
+
+// Define config keys to avoid 'any' type index errors
```

Comment:
sorry but I really don't think this is helpful :p Let's follow our project convention for now

Reply:
Agreed! Way too many comments in this test, we need them sometimes imo but much fewer!